### PR TITLE
[physx] Update to 5.x

### PR DESCRIPTION
**Pull request by BillyONeal**
physx -> /dev/null
omniverse-physx-sdk -> physx

See discussion in https://github.com/microsoft/vcpkg/pull/32432/

### CODE REVIEW

File: ports/physx/vcpkg.json [hunk 1]
```diff
@@ -1,15 +1,22 @@
 {
   "name": "physx",
-  "version": "4.1.2",
-  "port-version": 6,
-  "description": "The NVIDIA PhysX SDK is a scalable multi-platform physics solution supporting a wide range of devices, from smartphones to high-end multicore CPUs and GPUs",
-  "homepage": "https://github.com/NVIDIAGameWorks/PhysX",
-  "license": null,
-  "supports": "!mingw & !(windows & arm)",
+  "version": "5.2.1",
+  "description": "The NVIDIA PhysX SDK is a scalable multi-platform physics solution supporting a wide range of devices, from smartphones to high-end multicore CPUs and GPUs. This is the latest NVIDIA official version of the PhysX engine which also directly integrates into Omniverse. This port differs from the other 'physx' port (pointing to a repo no longer officially supported by NVIDIA) in supported platforms. It guarantees support only for Omniverse supported platforms. This port is NOT officially supported by NVIDIA.",
```
Comment (BillyONeal) [inline ports/physx/vcpkg.json:4]:
```suggestion
  "description": "The NVIDIA PhysX SDK is a scalable multi-platform physics solution supporting a wide range of devices, from smartphones to high-end multicore CPUs and GPUs. This is the latest NVIDIA official version of the PhysX engine which also directly integrates into Omniverse. This port is NOT officially supported by NVIDIA.",
```

**Review by FrankXie05 — APPROVED** [review]

**Comment by marcodiiga** [comment]
Tested on x64 linux, works great. Fine for me 👍🏻 Ty!

**Comment by dg0yt** [comment]
Hm, this PR moved the unofficial cmake package names which were tailored for the `omiverse-physx` port to the `physx` port which should use a different set.

**Comment by korDen** [comment]
I think this broke includes. physx source code is using #include "foundation/PxQuat.h" which now fails because the right include is now "physx/foundation/PxQuat.h" and "foundation/PxQuat.h" is not found, which leads to compile errors.

**Comment by dg0yt** [comment]
@korDen You are probably using Visual Studio without CMake integration? Then you may need to manually add the proper physx include dir to your project.

**Pull request merged**